tests/php/View/RequirementsTest.php

```diff
@@ -1109,4 +1110,77 @@ protected function getBackendFiles($backend, $type)
         }
         return array();
     }
+
+    public function testAddI18nJavascript()
+    {
+        /** @var Requirements_Backend $backend */
+        $backend = Injector::inst()->create(Requirements_Backend::class);
+        $this->setupRequirements($backend);
+        $backend->add_i18n_javascript('i18n');
+
+        $actual = $backend->getJavascript();
+
+        // English and English US should always be loaded no matter what
+        $this->assertArrayHasKey('i18n/en.js', $actual);
+        $this->assertArrayHasKey('i18n/en_US.js', $actual);
+        $this->assertArrayHasKey('i18n/en-us.js', $actual);
```

Comment:
Why is there 2 of these?

Reply:
Are regular translations are in the `en_US.js` format. MomentJS Locale files use the `en-us.js` format.